resources/views/admin/scholarship/partials/_form_scholarship.blade.php

```diff
@@ -157,8 +164,8 @@
 
 {{-- Recommendation Optional question --}}
 <div class="form-group">
-  {!! Form::label('display_optional_rec_question', 'Display optional recommendation question? ') !!}
-  {!! Form::checkbox('display_optional_rec_question', 1, ['class' => 'form-control']) !!}
+  {!! Form::label('display_optional_rec_question', 'Display optional recommendation question') !!}
+  {!! Form::checkbox('display_optional_rec_question', 1, $scholarship->display_optional_rec_question) !!}
```

Comment:
Do you still need to pass the `form-control` class to these checkboxes or no?

Reply:
ah, I commented on that [here](https://github.com/DoSomething/longshot/pull/881#discussion_r136380779) but it got collapsed

Reply:
Ah ok. cool. So not vital for merge, but I think in general checkboxes are better aligned on the left, with the label to the right of it. Especially if say you have a couple in a series, it'd look all weird if the labels were different lengths and the checkboxes came after the description on the right. But totally can be an update in a later PR and if you feel it's a good suggestion or not :)

Reply:
@weerd great suggestion!! I went ahead and rolled it in here since it was real quick!